spec/models/piece_spec.rb

```diff
@@ -225,8 +225,7 @@
       expect(king.y_position).to eq 3
     end
 
-    # it 'should move the piece to the destination square; ' +
-    #     'if the square is occupied, mark that piece as captured' do
+    # it 'should move the piece to the destination square; if the square is occupied, mark that piece as captured' do
```

Comment:
hey @kyidkim i think it may be a better idea to totally delete this test rather than comment part of it out.  otherwise this PR looks good!